installer/installer.go

```diff
@@ -208,6 +208,11 @@ func (i *Installer) GetArtifactName() string {
 	return i.ar.GetArtifactName()
 }
 
+// Returns the artifact provides header info field for artifact version >= 3.
+func (i *Installer) GetArtifactProvides() *artifact.ArtifactProvides {
+	return i.ar.GetArtifactProvides()
```

Comment:
Can you switch to `i.ar.MergeArtifactProvides()`? This will give you the [global](https://github.com/mendersoftware/mender-artifact/blob/master/Documentation/artifact-format-v3.md#artifact_provides) and [payload specific](https://github.com/mendersoftware/mender-artifact/blob/master/Documentation/artifact-format-v3.md#artifact_provides-1) (`type-info`) provides merged, as implemented in [MEN-2549](https://tracker.mender.io/browse/MEN-2549).

Reply:
Good catch! I've completely overseen the `type-info` part of the artifact format.